PageSpeed/MainViewController.swift

```diff
@@ -12,23 +12,23 @@ import Moya
 // MARK: - MainTabBarController
 
 class MainTabBarController: UITabBarController {
-
+    
```

Comment:
@ilyageekhub here is `Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)`

Reply:
fix